docs/getting-started/release-notes/v2_11_0.md

```diff
@@ -47,17 +48,16 @@ Zowe Version 2.11.0 contains the bug fixes that are described in the following t
 
 #### Zowe CLI (Core)
 
-#### Zowe CLI Imperative Framework
-
-#### CICS Plug-in for Zowe CLI
-
-#### DB2 Plug-in for Zowe CLI
+- Fixed an issue in the Daemon server which prevents users on Windows with uppercase letters in their username from using the Daemon.
+- Add check for invalid block size when creating a sequential data set. ([#1439](https://github.com/zowe/zowe-cli/issues/1439))
```

Comment:
@adam-wolfe what does the new "check" actually do? does it prevent the user from doing something? does it warn the user that something is wrong? this note could use some clarity.

Reply:
It prevents a user from allocating a data set with an invalid block size.

In more detail:

If the user issues a Zowe CLI command to create a sequential data set with a variable blocked record format and **does not specify a block size**, Zowe CLI will calculate and use a valid block size based on the logical record length.

If the user issues a Zowe CLI command to create a sequential data set with a variable blocked record format and **specifies a block size that is too small**, Zowe CLI will calculate and use a valid block size based on the logical record length.